src/index.js

```diff
@@ -11,14 +11,20 @@ export function configure() {
   const attached = Controller.prototype.attached;
   const detached = Controller.prototype.detached;
 
+  key.filter = () => true; // disable Keymaster's built-in filter
```

Comment:
Remove duplicated defaultKeymasterFilter, just get it from keymaster.

```js
const defaultKeymasterFilter = key.filter;
key.filter = () => true;
```

Reply:
Since we have local copy of keymaster, the other way we can clean it up is to totally remove filter from keymaster file, then use your code here.